python/FluidSolver.py

```diff
@@ -88,7 +88,7 @@
 t = 0
 
 print("Fluid: init precice...")
-precice_tau = interface.initialize()
+precice_dt = interface.initialize()
```

Comment:
Maybe we should add a comment here: `# precice defines timestep size of solver via precice-config.xml`